environment/xiaomi-xiaofang-t20l.uenv.txt

```diff
@@ -7,6 +7,7 @@ gpio_mmc_power=48o
 gpio_wlan=62O
 gpio_led_y=38
 gpio_led_b=39
+gpio_reset_button=46
```

Comment:
parameter for reset button is just `gpio_button`. please correct and i'll merge the pr. thank you.

Reply:
The list of available parameters
https://github.com/themactep/thingino-firmware/wiki/Camera-configuration#bootloader-environment-explained